### ISSUE TITLE
delete/remove photos

### ISSUE THREAD

**Issue by s4069b** [tbar0970/jethro-pmm#602]
am I correct in thinking there is no easy way to delete or remove a photo from a family or a person?

**Comment by tbar0970** [comment]
Correct.  Best workaround currently is to upload some sort of blank placeholder.
Adding to feature request list.

**Comment by jefft** [comment]
Another request for this.

**Comment by radarhere** [comment]
I've created https://github.com/tbar0970/jethro-pmm/pull/1187
